### ISSUE TITLE
What are the benefits of this driver

### ISSUE THREAD

**Issue by AnthonyPhan** [WCHSoftGroup/ch34xser_macos#30]
Hi all,

I have a CH40E board connected to my MacBook Air for serial communications and all seems to work well using the standard driver that ship with MacOS. Is there any benefits to installing the official drivers provided by WCH Soft Group if the default drivers already support the device? for example better board rate support?

**Comment by kiler129** [comment]
You are right, it *seems* to work. But without this driver a lot of things are broken, e.g. https://github.com/espressif/esptool/issues/908#issuecomment-2257272462
